CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## Next
 
+### Changed
```

Comment:
Het label van het issue is `feature`. Dat vertaalt zich naar `Added` in de `CHANGELOG.md`